CHANGES_tcell.md

```diff
@@ -0,0 +1,56 @@
+# Change from termbox to tcell
+
+Original GOCUI was written on top of [termbox](https://github.com/nsf/termbox-go) package. This document describes changes which were done to be able to use to [tcell/v2](https://github.com/gdamore/tcell) package.
+
+## Attribute color
+
+Attribute type represents a terminal attribute like color and font effects. Color and font effects can be combined using bitwise OR (`|`).
+
+In `termbox` colors were represented by range 1 to 256. `0` was default color which uses the terminal default setting.
+
+In `tcell` colors can be represented in 24bit, and all of them starts from 0. Valid colors have special flag which gives them real value starting from 4294967296. `0` is a default similart to `termbox`.    
+The change to support all these colors was made in a way, that original colors from 1 to 256 are backward compatible and if user has color specified as
+`Attribute(ansicolor+1)` without the valid color flag, it will be translated to `tcell` color by subtracting 1 and making the color valid by adding the flag. This should ensure backward compatibility.
+
+All the color constants are the same with different underlying values. From user perspective, this should be fine unless some arithmetic is done with it. For example `gocui.ColorBlack` was `1` in original version but is `4294967296` in new version.
+
+GOCUI provides a few helper functions which could be used to get the real color value or to create a color attribute.
+
+- `(a Attribute).Hex()` - returns `int32` value of the color represented as `Red << 16 | Green << 8 | Blue`
+- `(a Attribute).RGB()` - returns 3 `int32` values for red, green and blue color.
+- `gocui.GetColor(string)` - creates `Attribute` from color passed as a string. This can be hex value or color name (W3C name).
+- `gocui.Get256Color(int32)` - creates `Attribute` from color number (ANSI colors).
+- `gocui.GetRGBColor(int32)` - creates `Attribute` from color number created the same way as `Hex()` function returns.
+- `gocui.NewRGBColor(int32, int32, int32)` - creates `Attribute` from color numbers for red, green and blue values.
```

Comment:
```suggestion
- `(a gocui.Attribute).Hex()` - returns `int32` value of the color represented as `Red << 16 | Green << 8 | Blue`
- `(a gocui.Attribute).RGB()` - returns 3 `int32` values for red, green and blue color.
- `gocui.GetColor(string)` - creates `gocui.Attribute` from color passed as a string. This can be hex value or color name (W3C name).
- `gocui.Get256Color(int32)` - creates `gocui.Attribute` from color number (ANSI colors).
- `gocui.GetRGBColor(int32)` - creates `gocui.Attribute` from color number created the same way as `Hex()` function returns.
- `gocui.NewRGBColor(int32, int32, int32)` - creates `gocui.Attribute` from color numbers for red, green and blue values.
```